src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3617,6 +3810,9 @@ class CgroupUtils(object):
             if devclass == 'mic' or devclass == 'gpu':
                 for instance in devices[devclass]:
                     dev = devices[devclass][instance]
+                    if 'extra_devs' in dev:
+                        for extra in dev['extra_devs']:
+                            accelerators.append('%s' % extra)
```

Comment:
Any reason you didn't just append extra?  or str(extra)?

Reply:
I can actually just use extend.